neuralforecast/models/gru.py

```diff
@@ -71,6 +73,7 @@ def __init__(
         inference_input_size: int = -1,
         encoder_n_layers: int = 2,
         encoder_hidden_size: int = 200,
+        encoder_activation: str = "tanh",
```

Comment:
This should now be:

`encoder_activation: Optional[str] = None`

The default value is now None, so that the deprecation message doesn't show by default. Whenever somebody uses the old input "tanh", the deprecation message is shown.

The way it currently goes is that the deprecation warning is always shown, regardless of whether the argument is actively used by someone.

Reply:
Done